assets/js/components/channels/DownChannelsStatus.jsx

```diff
@@ -12,7 +13,9 @@ class DownChannelsStatus extends PureComponent {
   render() {
     const { channels, timestamp } = this.props
     const channelNames = map(channels, (channel) => channel.name)
-    return this.downChannelsDescription(channelNames, timestamp)
+    const paused = every(channels, (channel) => channel.statusInfo && channel.statusInfo.status == "paused")
```

Comment:
```suggestion
    const paused = every(channels, (channel) => channel.statusInfo?.status == "paused")
```
[Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)